### DISCUSSION TITLE
29. Eviter les animations

### DISCUSSION THREAD

**Discussion by dNicolle** [cnumr/best-practices-wordpress#37] [Bonnes pratiques] [answered]
https://github.com/cnumr/fiches-wordpress-greenit/blob/main/best-practices-WordPress/fiches/29.%20Eviter%20les%20animations.md

**Comment by dNicolle** [comment]
@Yanngautreau  La structure du format de la liste à puce semble pété
Tu veux de l'aide pour les principes de validations ?

**Comment by Yanngautreau** [comment]
j'ai modifié, c'est mieux ?

**Comment by florinesueur** [comment, marked as answer]
BP review
Issue : https://github.com/cnumr/fiches-wordpress-greenit/issues/82
